src/runtime_src/core/edge/skd/xrt_skd.cpp

```diff
@@ -338,7 +341,9 @@ namespace xrt {
    */
   int skd::deleteSoftKernelFile()
   {
-    return(remove(sk_path));
+      if(std::filesystem::exists(sk_path))
+	return(remove(sk_path));
+      return 0;
```

Comment:
Use std::filesystem API for removing file.

Reply:
Do not use `()` for `return`